slippi/default.nix

```diff
@@ -42,13 +66,27 @@ stdenv.mkDerivation rec {
     '';
 
   installPhase = if playbackSlippi then ''
+    wrapProgram "$out/dolphin-emu" \
+      --set "GDK_BACKEND" "x11" \
+      --prefix GIO_EXTRA_MODULES : "${glib-networking}/lib/gio/modules" \
+      --prefix LD_LIBRARY_PATH : "${vulkan-loader}/lib" \
+      --add-flags '-u $HOME/.config/slippi-playback'
     ln -s $out/dolphin-emu $out/bin/slippi-playback
+    ln -s ${playback-desktop}/share/applications $out/share
   '' else ''
+    wrapProgram "$out/dolphin-emu" \
+      --set "GDK_BACKEND" "x11" \
+      --prefix GIO_EXTRA_MODULES : "${glib-networking}/lib/gio/modules" \
+      --prefix LD_LIBRARY_PATH : "${vulkan-loader}/lib" \
+      --add-flags '-u $HOME/.config/slippi-netplay'
```

Comment:
I mentioned setting the configuration directory to a variable earlier. This choice of picking a directory explicitly (`$HOME/.config/slippi-netplay`) seems better, and closer to XDG-spec 👍🏽